pkg/ddl/ingest/backend_mgr.go

```diff
@@ -144,7 +144,7 @@ func genJobSortPath(jobID int64, checkDup bool) (string, error) {
 }
 
 // CreateLocalBackend creates a local backend for adding index.
-func CreateLocalBackend(ctx context.Context, store kv.Storage, job *model.Job, checkDup bool, adjustedWorkerConcurrency int) (*local.BackendConfig, *local.Backend, error) {
+func CreateLocalBackend(ctx context.Context, store kv.Storage, job *model.Job, checkDup bool, adjustedWorkerConcurrency int, globalSort bool) (*local.BackendConfig, *local.Backend, error) {
```

Comment:
I think `job.ReorgMeta` has already contained this information?

Reply:
[`91f225e` (#60848)](https://github.com/pingcap/tidb/pull/60848/commits/91f225ee083d69d5b9e1d6b08fab3013e199957b)